extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeMusicSearchExtractor.java

```diff
@@ -301,16 +294,7 @@ public String getUploaderUrl() throws ParsingException {
                                         .getObject(1).getObject("musicResponsiveListItemFlexColumnRenderer")
                                         .getObject("text").getArray("runs").getObject(0).getObject("navigationEndpoint");
 
-                                if (navigationEndpoint == null) {
-                                    return null;
-                                }
-
-                                final String url = getUrlFromNavigationEndpoint(navigationEndpoint);
-
-                                if (url != null && !url.isEmpty()) {
-                                    return url;
-                                }
-                                throw new ParsingException("Could not get uploader url");
+                                return getUrlFromNavigationEndpoint(navigationEndpoint);
```

Comment:
`getUrlFromNavigationEndpoint(navigationEndpoint)` returns null when navigationEndpoint is an empty Object or YouTube changed something in their JSON. In this case, we could not get the uploader URL and should throw a ParsingException. So please keep the null check for the URL.